prep-html-script-tag/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script src="main.js">
+       console.log('I have added JS to a script tag in the HTML!');
+       console.log('I have added a script file via the src attribute!');
```

Comment:
this line of code should be in the main.js file, that is what the script tag with the `src` attribute it for, to load an external sheet